test/sort-array.test.js

```diff
@@ -0,0 +1,16 @@
+import test from 'ava'
+import {sortArray} from '../src'
```

Comment:
Please add your `sortArray` method to the [`src/index.js`](https://github.com/eggheadio-github/stack-overflow-copy-paste/blob/038dc6d7065c1550506c80d6d7b3743132343be6/src/index.js) file. This is why [the tests are failing](https://travis-ci.org/eggheadio-github/stack-overflow-copy-paste/builds/128878783).